Makefile

```diff
@@ -13,10 +13,17 @@ include Makefile.global
 
 all: extension
 
+#build columnar only
+columnar:
+	$(MAKE) -C src/backend/columnar all
+	cp $(citus_abs_srcdir)/src/backend/columnar/columnar.so $(citus_abs_srcdir)/src/backend/distributed/
```

Comment:
Should be:

`cp $(citus_top_builddir)/src/backend/columnar/columnar.so $(citus_top_builddir)/src/backend/distributed/`